tests/should_succeed/bdd/proof.json

```diff
@@ -112,7 +112,7 @@
                         "tactic": "split_vc",
                         "children": [
                           { "prover": "cvc5@1.0.5", "time": 0.022 },
-                          { "prover": "cvc5@1.0.5", "time": 0.514 }
+                          { "prover": "cvc5@1.0.5", "time": 1.5 }
```

Comment:
Has this change really helped provers? As far as I can see the only impact it had was increasing the runtimes.

Reply:
I have examples which improve a lot, yes.

It's not very surprising that provers go slightly slower, because the goal is larger. But I don't expect them to fail where they succeed before.

Reply:
Also, my machine is quite slow, so it's not a good benchmark.